scripts/release_support/ova-release-notes/requirements.txt

```diff
@@ -16,4 +16,4 @@ requests==2.25.1
 requests-oauthlib==1.3.0
 requests-toolbelt==0.9.1
 six==1.15.0
-urllib3==1.26.3
+urllib3==1.26.8
```

Comment:
@gauravchaudhari02 Are we using this tool/script for generating OVA release notes. If not can we remove this codebase.
CC: @mukul-seagate11

Reply:
As discussed we will remove `requirements.txt` file itself